apps/tests/msg/rpmsg-flood-ping.c

```diff
@@ -92,7 +91,7 @@ static void rpmsg_name_service_bind_cb(struct rpmsg_device *rdev,
 		LPERROR("Unexpected name service %s.\r\n", name);
 	else
 		(void)rpmsg_create_ept(&lept, rdev, RPMSG_SERVICE_NAME,
-				       APP_EPT_ADDR, dest,
+				       RPMSG_ADDR_ANY, dest,
```

Comment:
Could you keep this test unchanged? the idea here is to keep one test with a fixed source address to also test a use case with a fixed source address.

Reply:
Ok